pyramid_log/__init__.py

```diff
@@ -53,6 +59,10 @@ class Formatter(logging.Formatter):
     determine the current request.
 
     '''
+    def __init__(self, *args, **kwargs):
+        kwargs['validate'] = False  # '.' not allowed in format strings by default
```

Comment:
The `validate` parameter was introduced in python 3.8.  In earlier python's its use will result in a `TypeError: unexpected keyword argument`.